src/app/app.ts

```diff
@@ -0,0 +1,75 @@
+import {Post} from 'mattermost-redux/types/posts';
+import {AppCall, AppState, AppCallResponse, AppCallValues} from 'mattermost-redux/types/apps';
+import {Tickets, CreatePayload} from 'node-zendesk';
+
+import mmClient from '../mattermost/client';
+import zendeskClient from '../zendesk/client';
```

Comment:
It'd be awesome to have cross-package imports be able to be absolute imports like this:

```suggestion
import mmClient from 'mattermost/client';
import zendeskClient from 'zendesk/client';
```

This would require changes to `tsconfig.json` for the typescript compiler to know where these are:

```json
{
    "compilerOptions": {
        "paths": {
            "mattermost/*": ["./src/mattermost/*"],
            "zendesk/*": ["./src/zendesk"],
            "src/*": ["./src/*"]
        }
    }
}

Reply:
I have been working on this for a while with no luck.  Will try a little bit more, but otherwise I'll create a ticket for this item.

Reply:
Created a ticket for this -> https://github.com/mattermost/mattermost-app-zendesk/issues/7